README.md

```diff
@@ -2,49 +2,42 @@
 
 _A tool to find ways to get involved on campus._
 
-## Getting Started
+## Contributing
 
-This project requires a working [Node.js](https://nodejs.org/en/) installation
-(at least v14) with NPM, which comes bundled with Node.js (at least v8). (If you
-use Yarn, that's fine, too. Just beware of the commands.)
+We are always open to contributions to the project. If you would like to contribute,
+please join our [Discord](http://discord.utdnebula.com/) and ask for how to access the Jupiter Dev Team
 
-First, clone the repository.
+Please make sure you have at least NodeJS v18.0.0 or greater installed before continuing.
```

Comment:
It might be worth linking to the standard setup/install docs for Node here

Reply:
Added link 👍🏽